force-app/main/default/lwc/miscI18n/miscI18n.html

```diff
@@ -0,0 +1,38 @@
+<template>
+    <lightning-card title="MiscI18n" icon-name="custom:custom19">
+        <div class="slds-var-m-around_medium">
+            <p>
+                User's locale:
```

Comment:
Add a space after colon for the better readability and grammatically correct (maybe we can use nbsp)

Reply:
Thanks Satya for your review, I will add the spaces and let you know.

Reply:
Changes made, please let me know.